src/qt/locale/bitcoin_it.ts

```diff
@@ -400,15 +400,15 @@
     </message>
     <message>
         <source>Encrypt the private keys that belong to your wallet</source>
-        <translation>Cifra le chiavi private che appartengono al tuo portamonete</translation>
+        <translation>Cripta le chiavi private che appartengono al tuo portafoglio</translation>
```

Comment:
Cripta le chiavi private del tuo portafoglio

Reply:
These are all the phrases we inherited from core when I swapped from portamonete to portafoglio, They do need some massaging... Thanks!